kebab/examples/create-bot/package-lock.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
you should probably switch all of the examples to yarn aswell while you're on it

Reply:
ok, I'll give it a go

Reply:
done (hopefully)